recipes/jplephem/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.5" %}
+
+package:
+  name: jplephem
+  version: {{ version }}
+
+source:
+  fn: jplephem-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/j/jplephem/jplephem-{{ version }}.tar.gz
+  md5: 2880f4a443cbc3ec3c9d90a9bc8e7d22
+
+build:
+  number: 0
```

Comment:
Add the following to `build`.

    script: python setup.py install --single-version-externally-managed --record record.txt

Then drop `build.sh`, but keep `bld.bat`.